app/models/invenio_rdm_record_converter.rb

```diff
@@ -611,4 +611,14 @@ def publisher
   def pediatric_neurology_brief?
     @generic_file.depositor == INSTITUTIONAL_PNB_DEPOSITOR || @generic_file.doi.shift.include?(PNB_DOI_PREFIX)
   end
+
+  def sizes
+    if @generic_file.id == "3105875f-61e1-412c-9b1c-c8a33b37ff35"
+      ["116 pages"]
+    elsif  @generic_file.id == "9cd9e3df-458c-4a2c-9e42-1dcdc95e7adf"
+      ["44 pages"]
+    elsif @generic_file.page_count
+      ["#{@generic_file.page_count} pages"]
```

Comment:
This is another issue with pages that we should fix: `@generic_file.page_count` is an array so the final string looks like:

`'["0"] pages'`

when it should be:

`'0 pages'`

(e.g., https://vtfsmghslrepo02.fsm.northwestern.edu/records/653g4-8m023)
Although these 0 pages shouldn't really be the norm actually - was `@generic_file.page_count.blank?` discarded for a specific reason?

Popping the page count from `page_count` should probably be enough.

Reply:
Oh, I think that was actually on the previous version. I was wondering why that was there.

Reply:
Okay, updated!